app/src/main/res/values/strings_account.xml

```diff
@@ -13,6 +13,9 @@
     <string name="account_error_user_not_found">User account not found.</string>
     <string name="account_error_unknown">Error logging in.</string>
     <string name="account_error_dialog_dismiss">OK</string>
+    <string name="account_error_no_internet">Please check your internet connection and try again.</string>
+    <string name="account_error_no_internet_title">No Internet Connection</string>
```

Comment:
let's change the name of these phrases to use `not_connected` to match the `LoginResponse.Error` they are displayed for. That way naming is consistent